L.mapbox.accessToken is no longer a const but instead a settable vari…

rhwilburn opened the pull request:
L.mapbox.accessToken is now set-able for left hand assignment as opposed
to being a const. This matches the mapbox-gl typescript definition now
and the intended behavior of mapbox (js).

This solves the issue raised here:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/15898

rhwilburn commented:
Not sure why my other facebook issue is being tied to this. Its not intentional

andy-ms commented on types/facebook-js-sdk/index.d.ts:
```diff
@@ -113,6 +113,7 @@ declare namespace facebook {
     interface ShareDialogParams extends DialogParams {
         method: 'share';
         href: string;
+		picture: string;
```
Mixes tabs and spaces.
Also, please make `facebook-js-sdk` changes in a separate pull request.

rhwilburn replied:
Thanks for the advice. I am a bit of a git newb.

rhwilburn commented:
I think I have fixed this by using these commands on my repo:

git reset --hard <original commit id of my orignal change for this issue only>
git push origin HEAD --force

I believe this should have fixed the other issue I fixed being included. Please let me know if I haven't got this right.

andy-ms commented:
Thanks!
